src/skibidi_orm/migration_engine/adapters/sqlite3_typing.py

```diff
@@ -10,6 +10,9 @@
 
 
 class SQLite3Typing:
+    """
+    Class defining the typing for SQLite3 database objects. It has to be an abstract class to avoid circular imports.
```

Comment:
it's not abstract, right? i don't think we have to talk about circular imports in docstrings, it's for the user